Source/Plugins/SyntheticBuilding/Test/StatsGenMDFTest.cpp

```diff
@@ -70,7 +70,7 @@ class StatsGenMDFTest
     {
       Texture::CalculateMDFData<float, LaueOpsType, ContainerType>(angles, axes, weights, odf, mdfValues, angles.size());
       err = StatsGen::GenMDFPlotData<float, LaueOpsType, ContainerType>(mdfValues, x, y, size);
-    } catch(std::runtime_error)
+    } catch(std::runtime_error& e)
```

Comment:
Unless you are editing the exception, keep references const.

```suggestion
    } catch(const std::runtime_error& e)
```